### PR TITLE
Close #7 create test data nv

### LINKED ISSUE #7: Test Data: NV, SUPPNV and AG
### Background Information

Add NV or XP test dataset (decide on naming!) to {admiralneuro}, to contain amyloid/tau PET imaging data

Note 1: this issue should be on hold until #6 is completed. Once we have a DM dataset, ensure to use the same patients for NV/XP.

Note 2: this test data will be moved to [{pharmaversesdtm}](https://github.com/pharmaverse/pharmaversesdtm/) after the first release of {admiralneuro}

### Definition of Done

- [ ] `data-raw` folder should contain new program `nv_neuro` that creates DM datasetDM test dataset added to {admiralneuro} package
 - [ ] `R/data.R` should be updated to document the new DM dataset (see [here](https://github.com/pharmaverse/admiral/blob/main/R/data.R) for an example file)

### PR DESCRIPTION
**Pull request by jwang-lilly**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -4,3 +4,16 @@
 #' @keywords datasets
 #' @family datasets
 "dm_neuro"
+
+
```
Comment (manciniedoardo) [inline R/data.R:8]: Could you please remove these two empty lines

File: data-raw/nv_neuro.R [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+library(tibble)
+library(dplyr)
+library(stringr)
+library(admiral)
+
+# Read input data ----
+
+data("dm_neuro")
+
```
Comment (manciniedoardo) [inline data-raw/nv_neuro.R:14]: Could you please limit to one blank line max across the program - thanks

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks @jwang-lilly! Looks like a great start.

For the failing spellcheck, you need to add these words to `inst/WORDLIST` if happy with them: 
image

For the failing roxygen test, you need to run `roxygen2::roxygenize('.', roclets = c('rd', 'collate', 'namespace'))` (as detailed in the check)

Additionally please could you revert the changes to the favicon - I don't think we want to change those files in this PR.

**Comment by manciniedoardo** [comment]
@pharmaverse/admiralneuro please could one or two of you review the actual dataset and confirm if happy with the contents, or if any changes are required.

**Review by Fanny-Gautier — COMMENTED** [review]
No other comments than the ones already provided by Edoardo. Other than that, looks good to me!
Maybe 1 question: I thought we wanted more than 2 visits? Currently derived "Baseline" and "Week 26" visits only.

File: R/data.R (hunk 1 above)
Comment (jwang-lilly) [reply R/data.R]: Done!

File: data-raw/nv_neuro.R (hunk 2 above)
Comment (jwang-lilly) [reply data-raw/nv_neuro.R:14]: Done.

**Comment by jwang-lilly** [comment]
@manciniedoardo, what do you mean the following?
"Additionally please could you revert the changes to the favicon - I don't think we want to change those files in this PR."
Go back to use pharmaverse icons instead of admiralneuro icon sets?
When you merged pull request 4, it is merged to main.  Pull request 7 is branched off main. Please advise!

**Comment by jwang-lilly** [comment]
@Fanny-Gautier, we can add another visit between baseline and visit 13, may be do that when we make changes following the dm_neuro update to incorporate observational study. Or use more visits for fluid biomarkers and cognitions.

**Review by meilinjiang — COMMENTED** [review]
I have reviewed and it looks great to me! One comment: Do we need a variable like "NVLOC" for the location of procedure results, such as cerebellum, neocortical composite etc?

**Comment by manciniedoardo** [comment]
> @manciniedoardo, what do you mean the following? "Additionally please could you revert the changes to the favicon - I don't think we want to change those files in this PR." Go back to use pharmaverse icons instead of admiralneuro icon sets? When you merged pull request 4, it is merged to main. Pull request 7 is branched off main. Please advise!

The favicons you made in #4 are totally fine, but currently in this PR, you are deleting those files - not sure why 

image

So my comment is  just to undo those deletions as we want to keep the favicons as they are

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
@jwang-lilly I think you committed some other files that shouldn't be merged to main:
- .Rprofile
- Thumbs.db
- man/Thumbs.db
- figures/Thumbs.db
- pkgdown/favicon/Thumbs.db
- renv folder

Please could you remove these and also revert the changes to the .Rbuildignore?

**Comment by jwang-lilly** [comment]
Will do a fresh pull request.
